<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Seeding</class>
 <widget class="QWidget" name="Seeding">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Seeding</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="ckb_autogetlink">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Lấy bài từ post đã quét</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Số bài muốn seeding:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spin_seedings">
          <property name="maximum">
           <number>999</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="txt_links">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="plainText">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Nhập url bài viết. Mỗi dòng một bài..</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ckb_like">
        <property name="text">
         <string>Like bài viết</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QCheckBox" name="ckb_comment">
          <property name="text">
           <string>Comment</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Nội dung:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_contents">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Cấu hình</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_2">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Thời gian nghỉ(s):</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QSpinBox" name="sp_idle_from">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="maximum">
           <number>999</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_6">
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>đến</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QSpinBox" name="sp_idle_to">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="maximum">
           <number>999</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>60</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_7">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Số luồng:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="sp_threads">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="btn_schedule_2">
          <property name="styleSheet">
           <string notr="true">    background-color: purple;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    padding: 6px;
	color: white</string>
          </property>
          <property name="text">
           <string>Lập lịch</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_runnow_2">
          <property name="styleSheet">
           <string notr="true">    background-color: green;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    padding: 6px;
	color: white</string>
          </property>
          <property name="text">
           <string>Chạy ngay</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_cancel_2">
          <property name="styleSheet">
           <string notr="true">    background-color: red;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    padding: 6px;
	color: white</string>
          </property>
          <property name="text">
           <string>Huỷ</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>layoutWidget</zorder>
  <zorder></zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ckb_autogetlink</sender>
   <signal>clicked(bool)</signal>
   <receiver>txt_links</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ckb_comment</sender>
   <signal>clicked(bool)</signal>
   <receiver>le_contents</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_cancel_2</sender>
   <signal>clicked()</signal>
   <receiver>Seeding</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>407</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_runnow_2</sender>
   <signal>clicked()</signal>
   <receiver>Seeding</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>407</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
